riscv/src/continuations/memory_merkle_tree.rs

```diff
@@ -88,15 +88,26 @@ impl<T: FieldElement, const N_LEAVES_LOG: usize, const WORDS_PER_PAGE: usize>
                 .or_default()
                 .push((index_within_page, value));
         }
+        updates_by_page
+    }
+
+    /// Applies updates, given an iterator of (memory address, value) pairs.
+    /// Memory addresses are assumed to be word-aligned.
+    #[allow(dead_code)]
```

Comment:
what's the dead code here?

Reply:
It's used in the test and I thought it might still be useful in the future. I can also move this to the test modules if you prefer!

Reply:
no it's fine, I was just wondering why it's marked as dead code. Shouldn't the compiler/clippy be fine with it since it's `pub`, even without the annotation?

Reply:
No, I think that's because the `risv::continuations::memory_merkle_tree` module is not visible to any other crate, so clippy knows it's not used.